voyager/modules/consensus/movement/src/main.rs

```diff
@@ -221,12 +245,45 @@ impl ConsensusModuleServer<ModuleData, ModuleCall, ModuleCallback> for ModuleSer
 
     #[instrument(skip_all, fields(chain_id = %self.ctx.chain_id))]
     async fn self_client_state(&self, height: Height) -> RpcResult<Value> {
+        let ledger_version = self
+            .ctx
+            .ledger_version_of_height(height.revision_height)
+            .await;
+
+        let vault_addr = self
+            .ctx
+            .get_vault_addr(
+                self.ctx.ibc_handler_address.0.get().clone().into(),
+                Some(ledger_version),
+            )
+            .await
+            .unwrap();
+
+        let table_handle = self
+            .ctx
+            .aptos_client
+            .get_account_resource(
+                vault_addr.into(),
+                &format!("0x{}::ibc::IBCStore", self.ctx.ibc_handler_address),
+            )
+            .await
+            .unwrap()
+            .into_inner()
+            .unwrap()
+            .data["commitments"]["handle"]
+            .clone()
+            .as_str()
+            .unwrap()
+            .to_owned();
+
         Ok(serde_json::to_value(movement::client_state::ClientState {
             chain_id: self.ctx.chain_id.to_string(),
             l1_client_id: self.ctx.l1_client_id.clone(),
             l1_contract_address: self.ctx.l1_settlement_address,
             l2_contract_address: self.ctx.ibc_handler_address,
-            table_handle: AccountAddress(Default::default()),
+            table_handle: AccountAddress(Hash::new(
```

Comment:
```suggestion
            table_handle: AccountAddress(H256::new(
```

Reply:
`AccountAddress` has the type `Hash` as the inner type im confused

Reply:
sorry forgot my other PR wasn't merged yet, disregard